go/host/p2p/p2p.go

```diff
@@ -267,3 +280,38 @@ func (p *p2pImpl) sendBytes(wg *sync.WaitGroup, address string, tx []byte) {
 func (p *p2pImpl) getSequencer() string {
 	return p.peerAddresses[0]
 }
+
+// status returns the current status of the p2p layer
+func (p *p2pImpl) status() *host.P2PStatus {
+	status := &host.P2PStatus{
+		FailedReceivedMessages: int64(0),
+		FailedSendMessage:      int64(0),
+		ReceivedMessages:       int64(0),
+	}
+
+	for _, hostGauge := range p.hostGauges {
+		for gaugeName, gauge := range hostGauge {
+			switch gaugeName {
+			case _receivedMessage:
+				status.ReceivedMessages = gauge.Value()
+			case _failedMessageRead:
+			case _failedMessageDecode:
+				status.FailedReceivedMessages += gauge.Value()
+			case _failedWriteSendMessage:
+			case _failedConnectSendMessage:
+				status.FailedSendMessage += gauge.Value()
+			}
+		}
+	}
+	return status
+}
+
+func (p *p2pImpl) incHostGaugeMetric(host string, gaugeName string) {
+	if _, ok := p.hostGauges[host]; !ok {
+		p.hostGauges[host] = map[string]gethmetrics.Gauge{}
+	}
+	if _, ok := p.hostGauges[host][gaugeName]; !ok {
+		p.hostGauges[host][gaugeName] = gethmetrics.NewRegisteredGauge(_failedMessageRead, p.metricsRegistry)
```

Comment:
is this correct? 
shouldn't it be:
`p.hostGauges[host][gaugeName] = gethmetrics.NewRegisteredGauge(gaugeName, p.metricsRegistry)`

Reply:
good spot ! Yes it should